packages/process/src/node/process-backend-module.ts

```diff
@@ -42,4 +43,8 @@ export default new ContainerModule(bind => {
             return child.get(TerminalProcess);
         }
     );
+
+    bind(MultiRingBuffer).toSelf().inTransientScope();
```

Comment:
Just out of curiosity, if the `MultiRingBuffer` is not a singleton, how can it give a "unique" ID for the readers when calling `#getReader()`? Could this be a problem, or it is just fine as is. Thank you!

Reply:
This is OK since there's 1 `MultiRingBuffer` per process, the ids are unique per `MultiRingBuffer` if ti was a singleton you would get all the terminals output into a single ring buffer which would give funny results :)